core/src/main/java/bio/terra/pearl/core/dao/study/StudyEnvironmentSurveyDao.java

```diff
@@ -45,6 +45,10 @@ public List<StudyEnvironmentSurvey> findAllByStudyEnvIdWithSurvey(UUID studyEnvI
         return studyEnvSurvs;
     }
 
+    public List<StudyEnvironmentSurvey> findAllStudyEnvsWithSurveyId(UUID surveyId) {
```

Comment:
This isn't returning StudyEnvironments, so I think you can simplify the name
```suggestion
    public List<StudyEnvironmentSurvey> findBySurveyId(UUID surveyId) {
```